src/Google.Api.Gax/Platform.cs

```diff
@@ -0,0 +1,150 @@
+﻿using System;
```

Comment:
Do you need to add the licensing disclaimer ?  "Licensed under the Apache License ... "